### PR TITLE
Added Minikube install using Winget in Windows

### PR DESCRIPTION
**Pull request by dipankardas011**
Signed-off-by: Dipankar Das <dipsonu10@hotmail.com>

modified:   content/en/docs/tasks/tools/install-kubectl-windows.md

Added command to install minikube using winget in windows

### CODE REVIEW

File: content/en/docs/tasks/tools/install-kubectl-windows.md [hunk 1]
```diff
@@ -79,14 +79,19 @@ If you have installed Docker Desktop before, you may need to place your `PATH` e
...
    {{% tab name="choco" %}}
    ```powershell
    choco install kubernetes-cli
    ```
    {{% /tab %}}
+   {{% tab name="winget" %}}
+   ```powershell
+   winget install -e --id Kubernetes.minikube
+   ```
+   {{% /tab %}}
```
Comment (sftim) [inline content/en/docs/tasks/tools/install-kubectl-windows.md:94]: We should either list Winget first (it seems to be the solution that Microsoft recommends), or list Chocolatey, Scoop and then Winget (alphabetical order).

File: content/en/docs/tasks/tools/install-kubectl-windows.md [hunk 2]
```diff
@@ -79,14 +79,19 @@ If you have installed Docker Desktop before, you may need to place your `PATH` e
 
 ### Install on Windows using Chocolatey or Scoop
 
-1. To install kubectl on Windows you can use either [Chocolatey](https://chocolatey.org) package manager or [Scoop](https://scoop.sh) command-line installer.
+1. To install kubectl on Windows you can use either [Winget](https://docs.microsoft.com/en-us/windows/package-manager/winget/) windows package manager or [Chocolatey](https://chocolatey.org) package manager or [Scoop](https://scoop.sh) command-line installer.
```
Comment (sftim) [inline content/en/docs/tasks/tools/install-kubectl-windows.md:82]:
```suggestion
 1. To install kubectl on Windows you can use either the
    [Winget](https://docs.microsoft.com/en-us/windows/package-manager/winget/) Windows
    package manager, the [Chocolatey](https://chocolatey.org) package manager, or the
    [Scoop](https://scoop.sh) command-line installer.
```

**Review by sftim — CHANGES_REQUESTED** [review]
It looks like this PR installs `minikube` if people select Winget. The page is about installing `kubectl`. Let's not recommend installing a different package - readers might not scrutinize the instructions to notice the switch.

File: content/en/docs/tasks/tools/install-kubectl-windows.md (hunk 1 above)
Comment (dipankardas011) [reply content/en/docs/tasks/tools/install-kubectl-windows.md:94]: Ok

**Comment by dipankardas011** [comment]
Thanks for suggesting
Hope you have a Good Day🙂

**Comment by dipankardas011** [comment]
So I have to make the user install kubectl instead of minikube ?

**Comment by dipankardas011** [comment]
So there is no way to install kubectl using winget

**Pull request closed without merging**